src/packages/media/media/section-view/media-section-view.element.ts

```diff
@@ -1,48 +1,79 @@
 import { UMB_MEDIA_COLLECTION_ALIAS } from '../collection/index.js';
-import { css, html, customElement } from '@umbraco-cms/backoffice/external/lit';
+import type { UmbCollectionBulkActionPermissions, UmbCollectionConfiguration } from '../../../core/collection/types.js';
```

Comment:
Looks like you are importing from another package here. Consider if something is missing in the index.ts file of that module, so you can import from the alias instead.

Reply:
Good spot! I've exported them in PR #1290.